packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.18+4
```

Comment:
Non-breaking behavior changes should be minor, not bugfix, version updates, as described in the semver spec, so this should be 0.6.19.

Reply:
Done.